build fails with lxml 4.3.4 using Python 3.8

Error:
```
$ pip3 install -U genrss
Defaulting to user installation because normal site-packages is not writeable
Collecting genrss
  Using cached genrss-1.0.3-py3-none-any.whl (5.3 kB)
Collecting pytz==2019.1
  Using cached pytz-2019.1-py2.py3-none-any.whl (510 kB)
Collecting lxml==4.3.4
  Using cached lxml-4.3.4.tar.gz (2.5 MB)
Building wheels for collected packages: lxml
  Building wheel for lxml (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-bjkwh9cm/lxml/setup.py'"'"'; __file__='"'"'/tmp/pip-install-bjkwh9cm/lxml/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-3__65f2l
       cwd: /tmp/pip-install-bjkwh9cm/lxml/
  Complete output (3312 lines):
  Building lxml version 4.3.4.
  Building without Cython.
  Using build configuration of libxslt 1.1.34
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.8
  creating build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/sax.py -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/ElementInclude.py -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/__init__.py -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/_elementpath.py -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/cssselect.py -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/usedoctest.py -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/doctestcompare.py -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/pyclasslookup.py -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/builder.py -> build/lib.linux-x86_64-3.8/lxml
  creating build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/__init__.py -> build/lib.linux-x86_64-3.8/lxml/includes
  creating build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/defs.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/_html5builder.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/soupparser.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/clean.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/__init__.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/ElementSoup.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/diff.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/_diffcommand.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/formfill.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/html5parser.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/usedoctest.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/_setmixin.py -> build/lib.linux-x86_64-3.8/lxml/html
  copying src/lxml/html/builder.py -> build/lib.linux-x86_64-3.8/lxml/html
  creating build/lib.linux-x86_64-3.8/lxml/isoschematron
  copying src/lxml/isoschematron/__init__.py -> build/lib.linux-x86_64-3.8/lxml/isoschematron
  copying src/lxml/etree.h -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/etree_api.h -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/lxml.etree.h -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/lxml.etree_api.h -> build/lib.linux-x86_64-3.8/lxml
  copying src/lxml/includes/xpath.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/etreepublic.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/xmlerror.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/schematron.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/xslt.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/xmlparser.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/relaxng.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/xinclude.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/__init__.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/uri.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/dtdvalid.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/xmlschema.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/htmlparser.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/tree.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/config.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/c14n.pxd -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/lxml-version.h -> build/lib.linux-x86_64-3.8/lxml/includes
  copying src/lxml/includes/etree_defs.h -> build/lib.linux-x86_64-3.8/lxml/includes
  creating build/lib.linux-x86_64-3.8/lxml/isoschematron/resources
  creating build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/rng
  copying src/lxml/isoschematron/resources/rng/iso-schematron.rng -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/rng
  creating build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl
  copying src/lxml/isoschematron/resources/xsl/XSD2Schtrn.xsl -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl
  copying src/lxml/isoschematron/resources/xsl/RNG2Schtrn.xsl -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl
  creating build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_svrl_for_xslt1.xsl -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_message.xsl -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_abstract_expand.xsl -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_skeleton_for_xslt1.xsl -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_dsdl_include.xsl -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/readme.txt -> build/lib.linux-x86_64-3.8/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
  running build_ext
  building 'lxml.etree' extension
  creating build/temp.linux-x86_64-3.8
  creating build/temp.linux-x86_64-3.8/src
  creating build/temp.linux-x86_64-3.8/src/lxml
  x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DCYTHON_CLINE_IN_TRACEBACK=0 -I/usr/include/libxml2 -Isrc -Isrc/lxml/includes -I/usr/include/python3.8 -c src/lxml/etree.c -o build/temp.linux-x86_64-3.8/src/lxml/etree.o -w
  src/lxml/etree.c: In function ‘__Pyx_InitCachedConstants’:
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228068:34: note: in expansion of macro ‘__Pyx_PyCode_New’
  228068 |   __pyx_codeobj__90 = (PyObject*)__Pyx_PyCode_New(2, 0, 2, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__89, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_etree_pyx, __pyx_n_s_getitem, 95, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__90)) __PYX_ERR(0, 95, __pyx_L1_error)
         |                                  ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228080:34: note: in expansion of macro ‘__Pyx_PyCode_New’
  228080 |   __pyx_codeobj__92 = (PyObject*)__Pyx_PyCode_New(3, 0, 3, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__91, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_etree_pyx, __pyx_n_s_setitem, 98, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__92)) __PYX_ERR(0, 98, __pyx_L1_error)
         |                                  ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228092:34: note: in expansion of macro ‘__Pyx_PyCode_New’
  228092 |   __pyx_codeobj__94 = (PyObject*)__Pyx_PyCode_New(2, 0, 2, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__93, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_etree_pyx, __pyx_n_s_delitem, 101, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__94)) __PYX_ERR(0, 101, __pyx_L1_error)
         |                                  ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228104:34: note: in expansion of macro ‘__Pyx_PyCode_New’
  228104 |   __pyx_codeobj__96 = (PyObject*)__Pyx_PyCode_New(2, 0, 2, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__95, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_etree_pyx, __pyx_n_s_contains, 104, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__96)) __PYX_ERR(0, 104, __pyx_L1_error)
         |                                  ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228116:34: note: in expansion of macro ‘__Pyx_PyCode_New’
  228116 |   __pyx_codeobj__98 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__97, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_etree_pyx, __pyx_n_s_len, 107, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__98)) __PYX_ERR(0, 107, __pyx_L1_error)
         |                                  ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228128:35: note: in expansion of macro ‘__Pyx_PyCode_New’
  228128 |   __pyx_codeobj__100 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__99, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_etree_pyx, __pyx_n_s_iter_2, 110, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__100)) __PYX_ERR(0, 110, __pyx_L1_error)
         |                                   ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228162:35: note: in expansion of macro ‘__Pyx_PyCode_New’
  228162 |   __pyx_codeobj__104 = (PyObject*)__Pyx_PyCode_New(2, 0, 6, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__103, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_etree_pyx, __pyx_n_s_register_namespace, 174, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__104)) __PYX_ERR(0, 174, __pyx_L1_error)
         |                                   ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228182:35: note: in expansion of macro ‘__Pyx_PyCode_New’
  228182 |   __pyx_codeobj__106 = (PyObject*)__Pyx_PyCode_New(0, 0, 0, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_xmlerror_pxi, __pyx_n_s_clear_error_log_2, 11, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__106)) __PYX_ERR(1, 11, __pyx_L1_error)
         |                                   ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228194:35: note: in expansion of macro ‘__Pyx_PyCode_New’
  228194 |   __pyx_codeobj__108 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__107, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_xmlerror_pxi, __pyx_n_s_copy, 191, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__108)) __PYX_ERR(1, 191, __pyx_L1_error)
         |                                   ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228206:35: note: in expansion of macro ‘__Pyx_PyCode_New’
  228206 |   __pyx_codeobj__110 = (PyObject*)__Pyx_PyCode_New(2, 0, 2, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__109, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_xmlerror_pxi, __pyx_n_s_receive, 197, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__110)) __PYX_ERR(1, 197, __pyx_L1_error)
         |                                   ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228218:35: note: in expansion of macro ‘__Pyx_PyCode_New’
  228218 |   __pyx_codeobj__112 = (PyObject*)__Pyx_PyCode_New(1, 0, 1, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__111, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_xmlerror_pxi, __pyx_n_s_copy, 287, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__112)) __PYX_ERR(1, 287, __pyx_L1_error)
         |                                   ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228230:35: note: in expansion of macro ‘__Pyx_PyCode_New’
  228230 |   __pyx_codeobj__114 = (PyObject*)__Pyx_PyCode_New(2, 0, 4, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__113, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_xmlerror_pxi, __pyx_n_s_filter_domains, 325, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__114)) __PYX_ERR(1, 325, __pyx_L1_error)
         |                                   ^~~~~~~~~~~~~~~~
  In file included from /usr/include/python3.8/compile.h:5,
                   from /usr/include/python3.8/Python.h:138,
                   from src/lxml/etree.c:97:
  /usr/include/python3.8/code.h:122:28: note: declared here
    122 | PyAPI_FUNC(PyCodeObject *) PyCode_New(
        |                            ^~~~~~~~~~
  src/lxml/etree.c:411:11: error: too many arguments to function ‘PyCode_New’
    411 |           PyCode_New(a, 0, k, l, s, f, code, c, n, v, fv, cell, fn, name, fline, lnos)
        |           ^~~~~~~~~~
  src/lxml/etree.c:228242:35: note: in expansion of macro ‘__Pyx_PyCode_New’
  228242 |   __pyx_codeobj__116 = (PyObject*)__Pyx_PyCode_New(2, 0, 4, 0, CO_OPTIMIZED|CO_NEWLOCALS, __pyx_empty_bytes, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_tuple__115, __pyx_empty_tuple, __pyx_empty_tuple, __pyx_kp_s_src_lxml_xmlerror_pxi, __pyx_n_s_filter_types, 335, __pyx_empty_bytes); if (unlikely(!__pyx_codeobj__116)) __PYX_ERR(1, 335, __pyx_L1_error)
```

### Environment

* OS: Linux Mint 20
* Python version: 3.8.5

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,6 +2,7 @@ language: python
 python:
   - '3.6'
   - '3.7'
+  - '3.8'
 install:
   - pip install -r requirements-test.txt
   - pip install -e .

--- a/README.md
+++ b/README.md
@@ -10,9 +10,24 @@ RSS generator for python
 
 ## Installing
 
+genrss using lxml library that required:
+
+* libxml2 version 2.9.2 or later.
+* libxslt version 1.1.27 or later.
+
+To install the required development packages of these dependencies on Linux systems,
+use your distribution specific installation tool, e.g. apt-get on Debian/Ubuntu:
+
+```bash
+sudo apt-get install libxml2-dev libxslt-dev
+```
+
+
 Install and update using pip:
 
-``pip install -U genrss``
+```bash
+pip install -U genrss
+```
 
 
 ## A Simple Example

--- a/setup.py
+++ b/setup.py
@@ -19,7 +19,7 @@ if __name__ == '__main__':
         license='MIT',
         packages=['genrss'],
         install_requires=[
-            'lxml==4.3.4',
+            'lxml>=4.3.4',
             'pytz==2019.1'
         ]
     )

